From deee98b0b2aaf42a78d61b0103268b48357cd0b0 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Fri, 28 Oct 2011 10:03:50 +0800
Subject: [PATCH 0505/2463] ENGR00160935 [MX6]Build out thermal driver

Build out thermal driver by default, since it will
cause system boot fail on the uncalibrated parts.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index a24c803..81f9dd6 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1895,7 +1895,7 @@ CONFIG_MXC_GPU_VIV=y
 #
 # ANATOP_THERMAL
 #
-CONFIG_ANATOP_THERMAL=y
+# CONFIG_ANATOP_THERMAL is not set
 
 #
 # File systems
-- 
1.7.7.4

